src/v0/destinations/klaviyo/transform.js

```diff
@@ -242,7 +242,7 @@ const trackRequestHandler = (message, category, destination) => {
   if (message.timestamp) {
     attributes.time = message.timestamp;
   }
-  payload.data = { type: 'event' };
+  payload.data = { type: 'event', unique_id: message.messageId };
```

Comment:
Couldn't find any doc as this version is deprecated.

Reply:
this is fine, this will anyway would fail if some uses it, we need to do code clean up.